<launch>

  <node pkg="flirtlib_ros" type="simulate_scans" name="simulate_scans"
        output="screen">
    <param name="num_lasers" value="1"/>
    <param name="map_file" value="$(find willow_maps)/willow-full-0.025.pgm"/>
    <param name="resolution" value="0.025"/>
  </node>

  <node pkg="flirtlib_ros" name="markers" type="markers" output="screen">
    <param name="num_markers" value="1"/>
  </node>

  <node pkg="flirtlib_ros" name="place_rec_test" type="place_rec_test"
        output="screen">
    <param name="map_file" value="$(find navigation_stage)/stage_config/maps/willow-full-0.05.pgm"/>
    <param name="resolution" value="0.05"/>
    <param name="ref_pos_resolution" value="2.0"/>
    <param name="ref_angle_resolution" value="1.0"/>
    <param name="x0" value="-10.0"/>
    <param name="x1" value="922.0"/>
    <param name="y0" value="-17.0"/>
    <param name="y1" value="930.0"/>
    <param name="num_matches_required" value="10"/>
    <param name="scan_db" value="flirtlib"/>
  </node>

</launch>